### DISCUSSION TITLE
Bots will not heal husk infection unless the patient is damaged by other vitality reducing afflictions.

### DISCUSSION THREAD

**Discussion by Crazyjr20** [FakeFishGames/Barotrauma#14504] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Bots appear to only cure husk infection as an affliction if the target is deemed injured, as husk infection does not cause any vitality damage until it's too late and uncurable medics will ignore it unless you also recieved sufficient damage from other sources, this includes station medics, making guards not get healed after being bit by any husks (usually players intentionally cheesing to be honest) as the bite wounds was not high enough to trigger the doctor.

https://github.com/user-attachments/assets/4e954591-e35c-4ee5-9a1f-0fe37f902247

### Reproduction steps

1 - Start a game
2 - Spawn or hire a medic (For testing I did both, but quick method is _"spawn medicaldoctor cursor"_
3 - Give medic order to heal (or leave them to idly do it)
4 - Give yourself husk infection _"giveaffliction huskinfection 50"_
5 - They will tell you injuries are too minor
6 - Give yourself any type of vitality reducing affliction more than a tiny amount (for testing I used blunt force trauma) _"giveaffliction blunttrauma 20"_
7 - Now they will heal you as you are "Injured"

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.5.9.1 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Crazyjr20** [comment]
I didn't record it, but I did test this with station npcs too, though I think in vanilla they'd run out of Calyxanide very fast anyway.

**Comment by SomeRandomNoobKekeke** [comment]
Another one for the piggy bank https://github.com/FakeFishGames/Barotrauma/discussions/14411
![BaroDev (wide)](https://github.com/user-attachments/assets/79cd5412-ec94-45c6-8d1c-6af34a20ea69)

**Comment by Regalis11** [comment]
This has been addressed in the latest update

**Reply by AnalogKnight** [reply]
Do bots cure husk infections in any case?
Sometimes we would not want bots to cure a husk infection, such as when the target has the husk gene.
Will the doctor avoid curing the target in such cases?

**Reply by Crazyjr20** [reply]
They dont cure it if you are tagged as immune 
`IgnoreTreatmentIfAfflictedBy="husktransformimmunity,husktransformimmunitytemporary"`
As long as any mods you are using follow this format you will be immune and not cured.

**Reply by AnalogKnight** [reply]
> They dont cure it if you are tagged as immune `IgnoreTreatmentIfAfflictedBy="husktransformimmunity,husktransformimmunitytemporary"` As long as any mods you are using follow this format you will be immune and not cured.

Good to know that, thanks.
